bosh-director/lib/bosh/director/blob_util.rb

```diff
@@ -23,7 +23,11 @@ def self.copy_blob(blobstore_id)
     end
 
     def self.replace_blob(old_blobstore_id, path)
-      blobstore.delete(old_blobstore_id) if blobstore.exists?(old_blobstore_id)
+      begin
+        blobstore.delete(old_blobstore_id)
+      rescue Bosh::Blobstore::BlobstoreError => e
```

Comment:
oops forgot to mention. let's make sure to log to debug about failure ignore. i believe we do similar thing in other places.